lib/components/overlays/launcher/compact_launcher_overlay.dart

```diff
@@ -41,18 +41,33 @@ class CompactLauncherOverlay extends ShellOverlay {
 class _CompactLauncherOverlayState
     extends ShellOverlayState<CompactLauncherOverlay> {
   final scrollController = ScrollController();
+  final snapshotController = SnapshotController();
+  final List<DesktopEntry> applications = [];
 
   @override
   Future<void> requestShow(Map<String, dynamic> args) async {
+    applications.clear();
+    applications.addAll(ApplicationService.current.listApplications());
+
+    applications.sort(
+      (a, b) => a.getLocalizedName(context.locale).toLowerCase().compareTo(
+            b.getLocalizedName(context.locale).toLowerCase(),
+          ),
+    );
+
     if (scrollController.hasClients) scrollController.jumpTo(0);
     controller.showing = true;
-    animationController.forward();
+    snapshotController.allowSnapshotting = true;
```

Comment:
i forgot to remove the code but basically the flow is set snapshot to true, animate, wait for it to end, set snapshot to false, same for reverse animation

Reply:
The widget will be made into a raster texture, which is far faster to render instead of a whole widget. The controller requires a SnapshotWidget to work properly tho

Reply:
Yeah I get that, but then why not just always set the value to true lol? The controller also isn't set to any particular widget. You'd have to use the `SnapShotWidget` widget for this to be taken into effect.

Reply:
You can't interact with a texture, the controller produces a single static texture when you set to true. Also ye, the widget for the controller is not present

Reply:
Ohhh, the controller creates a texture only when the value is set to true. I see I see.